message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java

```diff
@@ -562,7 +562,8 @@ public boolean isMessageViewable(Message message) {
 		String siteId = entityManager.newReference(message.getReference()).getContext();
 
 		if (unlockCheck(SECURE_READ_DRAFT, siteService.siteReference(siteId))) {
```

Comment:
Human would have noticed this if is redundant and should have been deleted

Reply:
idk there are times to leave dead code if you think it will help out other developers. even better would be a test.